semgrep/semgrep/ignores.py

```diff
@@ -0,0 +1,106 @@
+from dataclasses import dataclass
+from re import sub
+from typing import List
+from typing import Sequence
+from typing import Tuple
+
+import attr
+
+from semgrep.constants import COMMA_SEPARATED_LIST_RE
+from semgrep.constants import NOSEM_INLINE_RE
+from semgrep.error import Level
+from semgrep.error import SemgrepError
+from semgrep.output import logger
+from semgrep.output import OutputHandler
+from semgrep.rule_match import RuleMatch
+from semgrep.types import RuleMatchMap
+
```

Comment:
What is this module for? Please add a header that explains this module's intended role and scope for any future visitor.

examples:
* https://github.com/python/cpython/blob/1fc41ae8709e20d741bd86c2345173688a5e84b0/Lib/getopt.py#L1-L15
* https://github.com/python/cpython/blob/1fc41ae8709e20d741bd86c2345173688a5e84b0/Lib/gzip.py#L1-L6
* https://github.com/python/cpython/blob/1fc41ae8709e20d741bd86c2345173688a5e84b0/Lib/warnings.py#L1

Reply:
left some documentation